internal/util/column_map.go

```diff
@@ -111,3 +111,13 @@ func getColumnName(f *reflect.StructField, dbTag tag.Options) string {
 	}
 	return dbTag.Values()[0]
 }
+
+func shouldIgnoreField(dbTag tag.Options) bool {
+	if dbTag == "-" {
```

Comment:
Changing this back to `dbTag.Equals("-")` and use the `dbTag.IsEmpty()` on line 118 will fix the coverage issues.  Then I'll merge and release

Reply:
Done. I tossed the change on as an additional commit to avoid making the delta harder to review; please feel free to squash when merging.